doc/user-docs/ThirdPartyToolProfiles.md

```diff
@@ -78,6 +78,19 @@ Assuming that you've installed Git Bash into `C:/Program Files/Git`:
 }
 ````
 
+## Git Bash (WOW64)
+
+Assuming that you've installed Git Bash into `C:/Program Files (x86)/Git`:
+
+```json
+{
+    "name" : "Git Bash",
+    "commandline" : "C:/Program Files (x86)/Git/bin/bash.exe",
```

Comment:
```suggestion
    "commandline" : "C:/Program Files (x86)/Git/bin/bash.exe -li",
```

while you're here, would you mind fixing the other `bash.exe` to use `-li`? the git folks let us know that leaving `-li` off will cause subtle issues, and we keep running into them in this repository :smile:

Reply:
```suggestion
    "commandline" : "%ProgramFiles(x86)%/Git/bin/bash.exe -li",
```